gemrb/plugins/GUIScript/GUIScript.h

```diff
@@ -23,6 +23,11 @@
 
 // NOTE: Python.h has to be included first.
 
+// This definition needs to be here to keep mingw happy with standard python.org packages
+#if _MINGW32_
```

Comment:
```suggestion
#if __MINGW32__
```

Reply:
Whoops...

I'll fix that and try again.

Sometimes I do wonder if I should be allowed in the general vicinity of a computer